src/style.css

```diff
@@ -0,0 +1,20 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin: 0;
```

Comment:
As task description says
`Remove all the outer spacing for all the heading and paragraphs.`
Let's add rule for `p` as well